restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
+h1,h2,h3 {text-align: center;}
+p {text-align: left;}
+</style>
+<body>
+
+<div id="main">
+<h1 id="title"> Becks:
+</h1>
+
+ <p id="def"> 
+    /bɛks/ <i>noun</i> NORTHERN ENGLISH </p>
+     <p id="def"><em>
+        A friendly, curious, passionate (sometimes feisty) , strong, hard-working woman
+     </em></p>
+     <img src="images/IMG_2736.jpg" alt="Becks">
+     
+</div>
+<!--<div>
+<p></p>I love reading
+I love music
+I aspire to be a good mum
+I am a good listener
+I like using my brain -->
```

Comment:
Commenting should be included only to help another set of eyes or the developers eyes understand the code. If it is not necessary, remove to keep code clean and tidy.